codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -185,7 +195,14 @@ public final T build(Http2Connection connection) {
                 reader = new Http2InboundFrameLogger(reader, frameLogger);
                 writer = new Http2OutboundFrameLogger(writer, frameLogger);
             }
+
+            // Configure the flow controllers.
+            connection.local().flowController(new DefaultHttp2LocalFlowController(connection, writer,
```

Comment:
I wonder if we can do this differently.  Seems like the way we have things structured right now the responsibility to create and set these to `Default` classes lies in the encoder/decoder.  Can we keep that decision in a central place?